src/promptflow/tests/test_configs/eager_flows/chat-basic/README.md

```diff
@@ -0,0 +1,117 @@
+# Basic chat
+A basic chat flow defined using class entry. It demonstrates how to create a chatbot that can remember previous interactions and use the conversation history to generate next message.
```

Comment:
no need to have this complex README here

Reply:
done.